packages/api/src/routes/medical/internal-hie.ts

```diff
@@ -42,4 +46,45 @@ router.get(
   })
 );
 
+/**
+ * POST /internal/hie/unlink
+ *
+ * Unlinks a patient from a facility and removes all data associated with it.
+ *
+ * @param req.query.patientId - The patient's ID.
+ * @param req.query.oid - The oid of the facility to unlink from.
+ * @param req.query.dryRun - If true, will only simulate the unlink operation.
+ */
+router.post(
+  "/unlink",
+  requestLogger,
+  asyncHandler(async (req: Request, res: Response) => {
+    const cxId = getUUIDFrom("query", req, "cxId").orFail();
+    const patientId = getUUIDFrom("query", req, "patientId").orFail();
+    const oid = getFrom("query").orFail("oid", req);
+    const dryRun = getFromQueryAsBoolean("dryRun", req);
+
+    const { log } = out(`unlinkPatientFromOrganization - patient ${patientId} - cxId ${cxId}`);
+
+    unlinkPatientFromOrganization({
+      cxId,
+      patientId,
+      oid,
+      dryRun,
+    }).catch(err => {
+      const msg = `Error unlinking patient from organization`;
+      log(`Error unlinking patient from organization: ${errorToString(err)}`);
+      capture.error(err, { extra: { msg, cxId, patientId, oid } });
```

Comment:
Pass the msg as first param and `error` on extra

Reply:
Also, NABD, but usually we have the error handling inside the command as a global try/catch, so the route is very clean and only calls the command.

Reply:
Yeah, there's no shortage of code that doesn't follow the most up-to-date patterns, or just was coded before we consolidated into a reference way of doing things. :)

We can leave as is, no worries, it just adds to the list of examples that go against the pattern - eventually it becomes the pattern if never addressed.